src/shared/components/post/post-listing.tsx

```diff
@@ -230,13 +230,13 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
   body() {
     const body = this.postView.post.body;
     return body ? (
-      <div className="col-12 card my-2 p-2">
+      <article id="postContent" className="col-12 card my-2 p-2">
```

Comment:
`id="postContent"` should be a class or a data-id or role etc., since ids should be unique within the page

Reply:
There is only one postContent in a post. This is the body text for a post:

![image](https://github.com/LemmyNet/lemmy-ui/assets/3112763/17fdbdf9-65a1-4eb3-93b4-4185b0365a1d)

Reply:
ahh this is on the post view page not on the lists page. My bad